packages/graphiql/cypress/plugins/index.js

```diff
@@ -11,9 +11,9 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-module.exports = (on) => {
-  on('task', require('@cypress/code-coverage/task'));
-  on('file:preprocessor', require('@cypress/code-coverage/use-babelrc'));
+module.exports = (_on, _config) => {
+  // on('task', require('@cypress/code-coverage/task'));
+  // on('file:preprocessor', require('@cypress/code-coverage/use-babelrc'));
```

Comment:
the `@cypress/code-coverage` stuff was broken for us, I can move this into another PR